apps/api/src/network/services/network/network.service.ts

```diff
@@ -0,0 +1,21 @@
+import { NodeHttpService } from "@akashnetwork/http-sdk";
+import { minutesToSeconds } from "date-fns";
+import { singleton } from "tsyringe";
+
+import { Memoize } from "@src/caching/helpers";
+import { GetNodesParams, GetNodesResponse } from "../../http-schemas/network.schema";
+
+@singleton()
+export class NetworkService {
+  constructor(private readonly nodeHttpService: NodeHttpService) {}
+
+  @Memoize({ ttlInSeconds: minutesToSeconds(5) })
+  async getNodes(network: GetNodesParams["network"]): Promise<GetNodesResponse> {
+    return await this.nodeHttpService.getNodes(network);
```

Comment:
@baktun14 what do you think if we just use here pre-built @akashnetwork/net package which contains all this information?

Reply:
maybe it's even possible to use that info on deploy-web. Our network doesn't change frequently and eventually we plan to switch to proxy